devel/ftnchek/Portfile

```diff
@@ -37,11 +37,22 @@ long_description \
                 function of the compiler. Prior to using ftnchek, the \
                 user should verify that the program compiles correctly.
 
+if {${os.major} >= 22} {
+   depends_build    port:groff
```

Comment:
Preferable to use `depends_build-append` instead, as in this way you know that you aren't resetting anything that might have been set prior.

Reply:
Good suggestion. Done with next commit.